session/session.go

```diff
@@ -201,7 +201,9 @@ func (manager *Manager) SessionDestroy(w http.ResponseWriter, r *http.Request) {
 	if err != nil || cookie.Value == "" {
 		return
 	}
-	manager.provider.SessionDestroy(cookie.Value)
+	// fix bug: cookie.Value has been urlencoded, so should be decoded here
+	sid, _ := url.QueryUnescape(cookie.Value)
+	manager.provider.SessionDestroy(sid)
```

Comment:
sid is use `hex.EncodeToString` and `hextable = "0123456789abcdef"`

So sid should only have these char. urlencode will change nothing